migration/sql-files/017-update-search-index.sql

```diff
@@ -0,0 +1,23 @@
+-- update support for Full Text Search Vector on work item description
+DROP TRIGGER IF EXISTS upd_tsvector ON work_items;
+DROP FUNCTION IF EXISTS workitem_tsv_trigger() CASCADE;
+DROP INDEX IF EXISTS fulltext_search_index;
+CREATE INDEX fulltext_search_index ON work_items USING GIN (tsv);
+
+UPDATE work_items SET tsv =
+    setweight(to_tsvector('english', id::text),'A') ||
+    setweight(to_tsvector('english', coalesce(fields->>'system.title','')),'B') ||
+    setweight(to_tsvector('english', coalesce(new.fields#>>'{system.description, content}','')),'C');
+
+CREATE FUNCTION workitem_tsv_trigger() RETURNS trigger AS $$
```

Comment:
Add a comment after this line that specifies what this trigger does and what it is for. If the comments is written inside the trigger it should be stored in the DB and we can always look it up using postgres board tool.